test/built-ins/Symbol/asyncIterator/prop-desc.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2015-2017 the V8 project authors. All rights reserved.
```

Comment:
This matches the conventions of the async function tests. I don't feel strongly about the name though.